netci.groovy

```diff
@@ -29,9 +27,7 @@ def osShortName = ['Windows 7' : 'win7',
                    'Windows_NT' : 'windows_nt',
                    'Ubuntu14.04' : 'ubuntu14.04',
                    'Ubuntu16.04' : 'ubuntu16.04',
-                   'Ubuntu16.10' : 'ubuntu16.10',
                    'Debian8.4' : 'debian8.4',
-                   'Fedora24' : 'fedora24',
```

Comment:
curious why this isn't replaced w/ 26 or 27?

Reply:
We could try to add Fedora27 here and have it directly reference the Helix queue instead of a Jenkins image, but I determined that it wouldn't be worth the effort since the jobs listed in this section are really only used for the readme.md badges and we'll be replacing them when https://github.com/dotnet/core-eng/issues/1901 is done anyways.

Reply:
It's possible I might have to add images for the new versions for 1.0/1.1 support anyways - in which case I'll toss them in here too - though I'd like to avoid that if at all possible.

Reply:
ah I did not realize this is about badging, I just saw it was a PR for master.  If the badging support is really critical please do let @Chrisboh know how you feel.

Reply:
The groovy file doesn't make it clear (I can fix that), but yes, they are used only for the badges and where they're explicitly requested in a PR (in which case they're redundant with the helix tests).